### ISSUE TITLE
verapdf should fail cleanly if no Java

### ISSUE THREAD

**Issue by gordonwoodhull** [quarto-dev/quarto-cli#14171]
We check for Java on installing verapdf, but not when running it.

So you can end up with a blank error if you uninstall Java after installing verapdf.

We should fail cleanly here to avoid confusion. (But only if we are on the JAR path and not verapdf from PATH or QUARTO_VERAPDF.)
